src/stactools/palsar/cog.py

```diff
@@ -26,14 +25,21 @@ def cogify(tile_path: str, output_directory: str):
     # Newer years (2019+) has xml file, ignore
     # Pre 2019, look for .hdr files, then remove hdr for actual file to use
     # for each valid file convert to cog
-    cogs = []
+    cogs = {}
```

Comment:
is there a good reason for this to be a dict rather than a list? I think the more-strict (IMHO) typing of a list of string is more in-keeping with the intent and offers more type safety.

Reply:
yes, I need the Band names for the Asset names in the STAC record. Which isn't in this branch, it's in stac-test & stac.